### PR TITLE
add navigation, fetch data, format all files, make it all look relatively good 

### PR DESCRIPTION
**Pull request by jellyfish-tom**
as in title, check vid

https://user-images.githubusercontent.com/433674/214559611-aa02cda2-1d86-441e-a4f2-15a0052a5c75.mov

### CODE REVIEW

File: src/components/WeatherListItem/utils.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import {BASE_URL} from '../../shared/consts';
+
+export const getImgSourceById = (id: string): string =>
+  `${BASE_URL}/img/w/${id}.png`;
+
+export const capitalizeFirsts = (text: string): string =>
+  text.replace(/(^\w{1})|(\s+\w{1})/g, letter => letter.toUpperCase());
```
Comment (adamTrz) [inline src/components/WeatherListItem/utils.ts:7]: Are those duplicates? 🤔

File: src/hooks/useWeather.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        id: citiesIds.join(','),
+        APPID: API_KEY,
+      });
+
+      const response = await api.get<GetWeatherResponse>(
+        `data/2.5/group?${queryParams}`,
+      );
+
+      setData(response.data?.list || []);
+    };
```
Comment (adamTrz) [inline src/hooks/useWeather.ts:24]: Please add some error handling and loading state as well :)

File: src/components/WeatherListItem/utils.ts (hunk 1 above)
Comment (jellyfish-tom) [reply src/components/WeatherListItem/utils.ts:7]: https://www.freecodecamp.org/news/how-to-capitalize-words-in-javascript/

image

Comment (adamTrz) [reply src/components/WeatherListItem/utils.ts:7]: That wasn't my question actually 😉 
I think I've seen this function in two places ;p

**Review by adamTrz — APPROVED** [review]

**Pull request merged**